contentcuration/contentcuration/migrations/0127_assessment_item_tuple_key_unique_constraint.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-12-09 05:33
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contentcuration', '0126_merge_20201207_2127'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='channel',
+            name='preferences',
+            field=models.TextField(default='{"license": null, "language": null, "author": null, "aggregator": null, "provider": null, "copyright_holder": null, "license_description": null, "mastery_model": "num_correct_in_a_row_5", "m_value": 5, "n_value": 5, "auto_derive_video_thumbnail": true, "auto_derive_audio_thumbnail": true, "auto_derive_document_thumbnail": true, "auto_derive_html5_thumbnail": true, "auto_derive_exercise_thumbnail": true, "auto_randomize_questions": true}'),
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='preferences',
+            field=models.TextField(default='{"license": null, "language": null, "author": null, "aggregator": null, "provider": null, "copyright_holder": null, "license_description": null, "mastery_model": "num_correct_in_a_row_5", "m_value": 5, "n_value": 5, "auto_derive_video_thumbnail": true, "auto_derive_audio_thumbnail": true, "auto_derive_document_thumbnail": true, "auto_derive_html5_thumbnail": true, "auto_derive_exercise_thumbnail": true, "auto_randomize_questions": true}'),
+        ),
```

Comment:
What are the above migrations for?

Reply:
@micahscopes could you remove the above two migrations from this migration file? It's an unfortunate consequence of our using `json.dumps` on a Python object so that the keys are not in a specified order, which means these migrations get regenerated even though there has been no change.

Reply:
Ah, yup, good catch!

Reply:
Sometimes I subconsciously avoid looking at big scary blobs of text like that